src/engine/LoadingScreen.cs

```diff
@@ -141,7 +176,15 @@ public override void _Process(float delta)
     {
         // Only elapse passed time if this is visible
         if (!Visible)
+        {
+            if (visible)
+            {
+                visible = false;
+                randomizeTipTimer.Stop();
+            }
+
             return;
+        }
 
         // Spin the spinner
         totalElapsed += delta;
```

Comment:
Or maybe it would be just simplest to add a "elapsedSinceTooltipChange" variable that is used here to handle the timing.